app/templates.go

```diff
@@ -139,3 +141,39 @@ const tooManyRequestsHTMLTemplate = `
   </body>
 </html>
 `
+
+func getCaptchaPage(formActionURL, siteKey string) string {
+	return fmt.Sprintf(`
+		<html>
+		  <head>
+			<title>Haaukins API</title>
+			<style>
+				body {
+					position: relative;
+				}
+				#myForm {
+					position: absolute;
+					top: 100px;
+					left: %s;
+					transform: translate(-%s, %s);
+					text-align: center;
+				}
+				.g-recaptcha {
+					display: inline-block;
+				}
+			</style>
+			<script src="https://www.google.com/recaptcha/api.js" async defer></script>
+		  </head>
+		  <body>
+			<form action="%s" method="POST" id="myForm">
+			  <div class="g-recaptcha" data-sitekey="%s" data-callback="verifyCaptcha"></div>
+			  <br/>
+			</form>
+			<script>
+				function verifyCaptcha() {
+					document.getElementById("myForm").submit();
+				}
+			</script>
+		  </body>
+		</html>	`, "50%", "50%", "50%", formActionURL, siteKey)
+}
```

Comment:
We needed for haaukins 😉

Reply:
Yeah i think we need it for haaukins too :D

Reply:
I have just created the issue on haaukins repo.